spec/components/topic_view_spec.rb

```diff
@@ -413,17 +413,33 @@
     context "#first_post_bookmark_reminder_at" do
       let!(:user) { Fabricate(:user) }
       let!(:bookmark1) { Fabricate(:bookmark_next_business_day_reminder, post: topic.first_post, user: user) }
+      let!(:bookmark2) { Fabricate(:bookmark_next_business_day_reminder, post: topic.posts[1], user: user) }
 
       it "gets the first post bookmark reminder at for the user" do
-        expect(TopicView.new(topic.id, user).first_post_bookmark_reminder_at).to eq_time(bookmark1.reminder_at)
+        topic_view = TopicView.new(topic.id, user)
+
+        bookmarked_posts = topic_view.bookmarked_posts
+        first = bookmarked_posts.first
```

Comment:
Super super trivial but:

Arguably `[0]` is more consistent with the `[1]` below it. Or, if you want to be fancy:

`first, second = bookmark_posts`

Reply:
> Arguably [0] is more consistent with the [1] below it.

Agree with this 👍, but this one: `first, second = bookmark_posts` is even better!